http-service/src/main/java/net/runelite/http/service/grandexchange/GrandExchangeController.java

```diff
@@ -0,0 +1,68 @@
+package net.runelite.http.service.grandexchange;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import net.runelite.http.service.grandexchange.osbuddy.GuidePriceResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/grand-exchange")
```

Comment:
I would probably call this `osb/ge` or something - it is not the grand exchange, but the OSB prices.